lti_consumer/lti_xblock.py

```diff
@@ -1026,6 +1026,15 @@ def lti_launch_handler(self, request, suffix=''):  # pylint: disable=unused-argu
 
         lti_consumer.set_custom_parameters(self.prefixed_custom_parameters)
 
+        for processor in self.get_parameter_processors():
+            try:
+                default_params = getattr(processor, 'lti_xblock_default_params', {})
+                lti_consumer.set_extra_claims(processor(self) or default_params)
```

Comment:
The original lost code did this:
```
                default_params = getattr(processor, 'lti_xblock_default_params', {})
                lti_parameters.update(default_params)
                lti_parameters.update(processor(self.xblock) or {})
```
I don't know if default_params can be a multi-key dict (it sounds like it can).  If default params has a key that the processor() result does not, then the old code would include that key, but the new code would not include it.  What is the correct behavior?

Reply:
I'm not too sure, but looking at the [readme doc](https://github.com/edx/xblock-lti-consumer/blob/master/README.rst#defining-an-lti-parameter-processor) I think the default params are to be used if the processor hits an exception (which should happen with the current logic as well). In any case, please let me know if it's better to explicitly include the default params anyway.

Reply:
This is now fixed! :+1: